phpBB/bin/phpbbcli.php

```diff
@@ -68,6 +68,11 @@
 $phpbb_container->get('request')->enable_super_globals();
 require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
 
+if (is_file($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php'))
+{
+	require_once($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php');
```

Comment:
It is in the database, so in theory it can be changed by the user, either via a database tool or via the restore functionality in the ACP.
Does it actually need to be configurable?